install-deps.sh

```diff
@@ -79,7 +80,7 @@ else
     debian|ubuntu|devuan)
         echo "Using apt-get to install dependencies"
         $SUDO apt-get install -y lsb-release devscripts equivs
-        $SUDO apt-get install -y dpkg-dev gcc
+        $SUDO apt-get install -y dpkg-dev gcc libcunit1-dev libaio-dev libssl-dev
```

Comment:
i believe libaio-dev and libssl-dev will be installed by `ceph-build-deps`. can we avoid building/running the unit tests of SPDK?

Reply:
sorry not get your idea, what is the relation between install libaio-dev／libssl-dev ad unit  tests?

Reply:
no relation. that's why i put a period between these two sentences.

Reply:
@tchaikov ok. I will remove libaio-dev  and libssl-dev to try. For spdk unit tests, they will be compiled when make. There was an option to disable compile ut, but they are removed by spdk now ... I don't want to add it in our fored spdk, otherwise, we have to maintain a seperate spdk version.